_config/config.yml

```diff
@@ -12,3 +12,9 @@ SilverStripe\Admin\LeftAndMain:
 SilverStripe\CMS\Model\SiteTree:
   extensions:
     - SilverStripe\CampaignAdmin\SiteTreeExtension
+
+# Allow users only only have access to the campaign-admin, and not all CMS access
+# i.e. CMS_ACCESS_LeftAndMain, to still be able to manage changesets
+SilverStripe\Versioned\ChangeSet:
```

Comment:
These used to be done in Versioned https://github.com/silverstripe/silverstripe-versioned/pull/482

The plural_name is used [here](https://github.com/silverstripe/silverstripe-cms/pull/3062/files#diff-8de4622cc87cca016e9e2d317482ab37ac8a29a900134ad8ffad3d89ee108937R1575) with a call to ChangeSet::singleton()->i18n_pluralise()

We may as well also keep the singlular_name for consistency